events/EiffelActivityDequeuedEvent/simple.json

```diff
@@ -0,0 +1,18 @@
+{
+  "meta": {
+    "type": "EiffelActivityDequeuedEvent",
+    "version": "1.0",
+    "time": 1234567890,
+    "domainId": "example.domain",
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+  },
+  "data": {
+    "reason": "The Job has been removed from the queue"
```

Comment:
I think the reason doesn't actually explain why. "The job has been obsoleted by another job and removed from the queue". Perhaps can be phrased better...